test_sets/API/vehicle_data.txt

```diff
@@ -36,3 +38,6 @@
 ./test_scripts/API/VehicleData/UnsubscribeVehicleData/006_UnsubscribeVD_2nd_request_IGNORED.lua
 ./test_scripts/API/VehicleData/UnsubscribeVehicleData/007_UnsubscribeVD_for_2_apps.lua
 ./test_scripts/API/VehicleData/UnsubscribeVehicleData/008_UnsubscribeVD_HMI_responds_with_not_success_codes.lua
+./test_scripts/API/VehicleData/UnsubscribeVehicleData/009_UnsubscribeVD_for_2_apps_different_data.lua
+./test_scripts/API/VehicleData/UnsubscribeVehicleData/010_UnsubscribeVD_for_2_apps_unregister_data.lua
+./test_scripts/API/VehicleData/UnsubscribeVehicleData/011_UnsubscribeVD_for_2_apps_unregister_different_data.lua
```

Comment:
These 3 scripts for `UnsubscribeVehicleData` have sequences that are not related to the original issue. Do we really need them?